pkg/api/definitions.go

```diff
@@ -47,3 +56,51 @@ func (tp *TestPlanDefinition) TestCaseByName(name string) (seq int, tc *TestCase
 	}
 	return -1, nil, false
 }
+
+func (tp *TestPlanDefinition) Describe(w io.Writer) {
+	p := func(w io.Writer, f string, a ...interface{}) {
```

Comment:
```suggestion
	print := func(w io.Writer, f string, a ...interface{}) {
```

tiny nit not worth changing but I felt sharing anyway :) having a word doesn't bite and saves a reader to have to look at the args to understand that it is for printing